nengo/processes.py

```diff
@@ -306,7 +306,7 @@ def make_step(self, size_in, size_out, dt, rng):
         presentation_time = float(self.presentation_time)
 
         def step_presentinput(t):
-            i = int(t / presentation_time + 1e-7)
+            i = int((t-dt) / presentation_time + 1e-7)
```

Comment:
Nit: I've had uses for this too. Can we add a constant for epsilon to a utility file like `nengo.utils.compat`? [One possibility](http://stackoverflow.com/questions/19141432/python-numpy-machine-epsilon) is `np.finfo(float).eps`?